### DISCUSSION TITLE
Will Zed ever add file event listeners to the extensions api?

### DISCUSSION THREAD

**Discussion by nezudevv** [zed-industries/zed#19091] [Feature Requests]
I'm looking at creating a few extensions but am not finding(via extension creation api) a good way to listen/track file changes (like opening or closing of a particular file). 

Is there a roadmap for adding more handlers to the extension api spec?

**Comment by JosephTLyons** [comment]
Thanks for reaching out. We want to better support our community, and the best way for us to do that is through our official Discord. The team is active there and happy to help.

We’re closing the “Help and General Q&A” category on GitHub Discussions and directing all support questions to Discord going forward.

https://zed.dev/support
